src/pages/loggedOutPage.tsx

```diff
@@ -0,0 +1,184 @@
+import { Grid, ThemeProvider, createTheme } from "@mui/material";
+import React from "react";
+import "../App.css";
+
+import loggedOutNav from "../components/loggedOut/loggedOutNavBar";
+import welcomeMessage from "../components/loggedOut/loggedOutWelcome";
+import advertSection from "../components/loggedOut/advertSection";
+import reviewsSection from "../components/loggedOut/reviewSection";
+
+export default function LoggedOutPage() {
+  const theme = pageTheme();
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Grid
+        container
+        sx={{
+          flexDirection: "column",
+        }}
+      >
+        {loggedOutNav()}
+        {/*-------------------------HEADER-------------------------*/}
+        <Grid
+          item
+          container
+          direction="column"
+          justifyContent="space-evenly"
+          p={5}
+          alignItems="center"
+          xs={12}
+          sx={{
+            background: "#0B7D66",
+            "@media (min-width: 600px)": {
+              flexDirection: "row",
+            },
+          }}
+        >
+          {/*-------------------------------Welcome Message--------------------------------------*/}
+
+          {welcomeMessage()}
+
+          {/*-------------------------------Login Bubble--------------------------------------*/}
+
+          <Grid item container xs={8} sm={6} md={4}></Grid>
+        </Grid>
+
+        {/*-----------------------Advertisement Section-------------------------------------*/}
+        {advertSection()}
+
+        {/*----------------------------Review Section-----------------------------------------*/}
+        {reviewsSection()}
+      </Grid>
+    </ThemeProvider>
+  );
+}
+/*theme for font sizes*/
+function pageTheme() {
```

Comment:
I fear that the theme will be common upon pages so we may need to make note for the future to reevaluate once we have things settled/started.

Reply:
For sure, i just wanted things to look more responsive for now at the very least